apple/MarkdownTextInputDecoratorShadowNode.mm

```diff
@@ -0,0 +1,255 @@
+            textInputProps.text, defaultTextAttributes});
+
+        auto attachments = BaseTextShadowNode::Attachments{};
+        BaseTextShadowNode::buildAttributedString(
+            defaultTextAttributes, *textInput, attributedString, attachments);
+
+        // convert the newly created attributed string to
+        // NSAttributedString
+        nsAttributedString = RCTNSAttributedStringFromAttributedStringBox(
+            AttributedStringBox{attributedString});
+      }
+    }
+
+    // apply markdown
+    auto newString = [utils parseMarkdown:nsAttributedString
+                withDefaultTextAttributes:defaultNSTextAttributes];
+
+    // create a clone of the old TextInputState and update the
+    // attributed string box to point to the string with markdown
+    // applied
+    newStateData.attributedStringBox =
+        RCTAttributedStringBoxFromNSAttributedString(newString);
+  } else if (stateData.attributedStringBox.getMode() ==
+             AttributedStringBox::Mode::OpaquePointer) {
+
+    // apply markdown
+    auto newString = [utils parseMarkdown:nsAttributedString
+                withDefaultTextAttributes:defaultNSTextAttributes];
+
+    // create a clone of the old TextInputState and update the
+    // attributed string box to point to the string with markdown
+    // applied
+    newStateData.attributedStringBox =
+        RCTAttributedStringBoxFromNSAttributedString(newString);
+  }
+
+  textInput->setStateData(std::move(newStateData));
+}
+
+// this is private in YogaLayoutableShadowNode
+YGSize MarkdownTextInputDecoratorShadowNode::yogaNodeMeasureCallbackConnector(
+    YGNodeConstRef yogaNode, float width, YGMeasureMode widthMode, float height,
+    YGMeasureMode heightMode) {
+
+  auto minimumSize = Size{0, 0};
+  auto maximumSize = Size{std::numeric_limits<Float>::infinity(),
+                          std::numeric_limits<Float>::infinity()};
+
+  switch (widthMode) {
+  case YGMeasureModeUndefined:
+    break;
+  case YGMeasureModeExactly:
+    minimumSize.width = floatFromYogaFloat(width);
+    maximumSize.width = floatFromYogaFloat(width);
+    break;
+  case YGMeasureModeAtMost:
+    maximumSize.width = floatFromYogaFloat(width);
+    break;
+  }
+
+  switch (heightMode) {
+  case YGMeasureModeUndefined:
+    break;
+  case YGMeasureModeExactly:
+    minimumSize.height = floatFromYogaFloat(height);
+    maximumSize.height = floatFromYogaFloat(height);
+    break;
+  case YGMeasureModeAtMost:
+    maximumSize.height = floatFromYogaFloat(height);
+    break;
+  }
+
+  // This is where changes begin compared to the copied code
+  const auto decoratorYogaNode = YGNodeGetParent(const_cast<YGNodeRef>(yogaNode));
+  const auto &decoratorShadowNode = shadowNodeFromContext(decoratorYogaNode);
+
+  LayoutContext context = LayoutContext();
+  context.fontSizeMultiplier = RCTFontSizeMultiplier();
+
+  auto size = decoratorShadowNode.measureContent(context, {minimumSize, maximumSize});
```

Comment:
Will it work?
```suggestion
  const auto size = decoratorShadowNode.measureContent(context, {minimumSize, maximumSize});
```

Reply:
[b555c40](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/b555c40a92f0c64b736584e6e5509f32f6ebc91d)